queries/python/function_arguments.scm

```diff
@@ -7,6 +7,12 @@
       (dictionary_splat_pattern
         (identifier) @argname)
     ]))
+(function_definition
+  parameters: (parameters
+    (default_parameter
+      [
+        (identifier) @argname
+      ])))
```

Comment:
This could be put after line 8 to avoid repeating `function_definition` and `parameters` ( `[]` in treesitter works as an "or")

Reply:
Thanks for the hint! This was my first time writing queries. I simplified the existing queries as well.